src/main/twirl/com/lightbend/lagom/docs/support.scala.html

```diff
@@ -0,0 +1,48 @@
+@import com.lightbend.lagom.docs.LagomContext
+
+@(implicit context: LagomContext)
+
+@page("Support") {
+
+
+    <h1>Support</h1>
+    <div class="small-12 columns">
+    <p><a target="_blank" href="https://lightbend.com">Lightbend</a> sponsors
```

Comment:
I think we should add the "www" to the URL, because otherwise it has to redirect.

Reply:
I'll fix.